api/lambda_function.py

```diff
@@ -11,8 +11,17 @@ def lambda_handler(event, context):
     request_type = event['requestType']
     store_id = event['storeId']
     customer_id = event['customerId']
-
-    response = {}
+    # write store_id & customer_id to the DynamoDB table using the object we instantiated and save response in a variable
+    response = table.put_item(
```

Comment:
I don't think we need this here because we don't want to insert into the DB every time the lambda is called. We want to handle based on the type of request we receive as we see in the if statement block below

Reply:
aah gotcha makes sense